extensions/fix-language/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "fix-language",
+  "title": "Fix Language",
+  "description": "Switch the language input source of the text when you type and forget to change the language.",
+  "icon": "command-icon.png",
+  "author": "tin5451",
+  "categories": [
+    "Productivity",
+    "Developer Tools",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "sw",
+      "title": "Sw",
+      "subtitle": "Quick switch the selected text input source",
+      "description": "Switch the language to another (default: eng/thai)",
+      "mode": "no-view"
+    },
+    {
+      "name": "swc",
+      "title": "Swc",
+      "subtitle": "Quick switch the clipboard text input source",
+      "description": "Switch the language to another (default: eng/thai)",
+      "mode": "no-view"
+    },
+    {
+      "name": "fix-language",
+      "title": "Fix Language",
+      "subtitle": "Switch the language input source",
+      "description": "Switch the language input source",
+      "mode": "view"
+    },
+    {
+      "name": "fix-language-setting",
+      "title": "Fix Language Setting",
```

Comment:
This should be moved to [preferences](https://developers.raycast.com/api-reference/preferences) instead

Reply:
I move it to [preferences](https://developers.raycast.com/api-reference/preferences) instead.